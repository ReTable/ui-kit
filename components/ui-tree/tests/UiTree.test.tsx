import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import { UiTree } from '~';

describe('UiTree', () => {
  it('works', () => {
    const tree = create(<UiTree />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
