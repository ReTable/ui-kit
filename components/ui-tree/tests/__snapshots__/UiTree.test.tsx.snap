// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiTree > works 1`] = `
<div
  className="tabula_ui_tree__1x3fa7k0 "
/>
`;
